src/updateConfig.ts

```diff
@@ -0,0 +1,246 @@
+import cp = require("child_process");
```

Comment:
can you add a comment that explains that this is a stand-alone script and that it's intended to make tslint.json have exactly the correct set of rule exemptions? (at least, that is what I think it does -- if I'm wrong, take that as something that definitely needs to be explained)

Reply:
see if the comment i added makes sense or if there's still some things that are unclear.